### DISCUSSION TITLE
Suggestion - Adding a "Download Filtered" function

### DISCUSSION THREAD

**Discussion by HowDoIOpenWord** [metafates/mangal#137] [Ideas]
Hey i'd like to propose a feature suggestion.  When filtering out a selection of chapters within a selected manga there's no choice of selecting all that are filtered.  The "tab select all" function selects all manga within the initial list provided by the API pull and not within the filtered list itself.

Could there be an additional key designated to select all within the filtered query?

**Comment by metafates** [comment]
Hello! I think it's possible, yes
